packages/website/components/account/storageManager/storageManager.js

```diff
@@ -102,29 +100,25 @@ const StorageManager = ({ className = '', content }) => {
                   standard: 'iec',
                 })}
               </span>
-              &nbsp;of <span className="storage-number">{maxSpaceLabel}</span> used
+              {currentPlan?.id === 'earlyAdopter' ? (
+                <>
+                  &nbsp;of <span className="storage-number">{maxSpaceLabel}</span> used
+                </>
+              ) : (
+                <>
+                  &nbsp;of <span className="storage-number">{currentPlan?.base_storage}</span> used
```

Comment:
in general IMO we should stick with camelCase like `baseStorage` instead of `base_storage`